root/static/scripts/edit/components/ReleaseMergeStrategy.js

```diff
@@ -40,6 +40,33 @@ const mergeStrategyOptions = {
   ],
 };
 
+// Please keep this in sync with root/static/scripts/release-editor/fields.js
```

Comment:
Seems like like we should just make it a utility function and call `hasUselessMediumTitle(self.name())` in the computed observable. :)